app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -80,7 +81,11 @@ public function store(Request $request, $seatId = null, $backTo = null)
             // Ooops.. something went wrong
             return redirect()->back()->withInput()->withErrors($validator);
         }
-        $return_to = User::find($licenseSeat->assigned_to);
+
+        if(($return_to = User::find($licenseSeat->assigned_to)) === null){
```

Comment:
Wouldn't this just be looking for a matching ID, not type? So if this was checked out to Asset ID 5, it would look for User ID 5 if it exists and would use that instead?

Reply:
No, `LicenseSeat->assigned_to` is used when assigned to a user, `LicenseSeat->asset_id` is used when assigned to an asset.